canonical-web-and-design/jaas.ai
# Add entity expert cards

## Done

- Add definitions of the experts to remove duplication.
- Display the expert cards on charm and bundle pages.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Visit / and /store
- You should see the expert cards as before.
- Visit /experts/spicule and /experts/tengu
- You should see the contact cards in the top right, just as before.
- Visit /u/spiculecharms/apache-drill and /u/spiculecharms/anssr-analytics/bundle/0 and /u/tengu-team/automate-spark-jobs/bundle/3
- You should see contact cards for the relevant experts.

## Details

- Fixes: #113.
